src/Node/Expression/PrefixUpdateExpression.php

```diff
@@ -13,7 +13,10 @@ class PrefixUpdateExpression extends UnaryExpression {
     /** @var Token */
     public $incrementOrDecrementOperator;
 
-    /** @var Variable */
+    /** 
+     * @phpstan-ignore-next-line Is not co-variant with overridden property
+     * @var Variable|Token 
+     */
     public $operand;
```

Comment:
not entirely sure if this `operand` accepts the same types as the parent `UnaryExpression`